pkg/service/servicefakes/fake_sipstore.go

```diff
@@ -146,34 +146,6 @@ type FakeSIPStore struct {
 		result1 *livekit.SIPTrunkInfo
 		result2 error
 	}
-	SelectSIPDispatchRuleStub        func(context.Context, string) ([]*livekit.SIPDispatchRuleInfo, error)
```

Comment:
why is this changing in this PR? Did some PR miss running `mage generate` before this?

Reply:
Yeah probably.

Reply:
I can remove the change from this PR if you want

Reply:
That's okay. Just make sure you are rebased to `master`

Reply:
Yeah I did that when I force pushed just before marking the PR ready for review.